pkg/operator/ceph/cluster/osd/spec_test.go

```diff
@@ -20,24 +20,24 @@ package osd
 import (
 	"testing"
 
-	cephv1 "github.com/rook/rook/pkg/apis/ceph.rook.io/v1"
-	rookalpha "github.com/rook/rook/pkg/apis/rook.io/v1alpha2"
-	"github.com/rook/rook/pkg/clusterd"
-	cephconfig "github.com/rook/rook/pkg/daemon/ceph/config"
-	"github.com/rook/rook/pkg/operator/ceph/cluster/osd/config"
+	//cephv1 "github.com/rook/rook/pkg/apis/ceph.rook.io/v1"
```

Comment:
Next step is to get the existing unit tests fixed. Besides some of these being commented out, others are failing.